ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/TopicStreamWriter.java

```diff
@@ -62,34 +56,13 @@ public static TopicStreamWriter create(
   ) {
     return new TopicStreamWriter(
         serviceContext.getSchemaRegistryClient(),
-        createTopicConsumer(
-            injectSupplierProperties(serviceContext, consumerProperties),
-            printTopic.getTopic().toString(),
-            printTopic.getFromBeginning()
-        ),
+        PrintTopicUtil.createTopicConsumer(serviceContext, consumerProperties, printTopic),
```

Comment:
nit: one parameter per line.

Reply:
I'm not personally a fan of extra line-breaks, but I've added them as you wish :)